jina/types/request/data.py

```diff
@@ -185,7 +241,7 @@ def parameters(self) -> Dict:
         :return: a Python dict view of the parameters.
         """
         # if u get this u need to have it decompressed
-        return json_format.MessageToDict(self.proto.parameters)
+        return json_format.MessageToDict(self.proto_wo_data.parameters)
```

Comment:
if I want to get the data, it is very weird that I access proto_wo_data

Reply:
Probably a bug in my solution then. I agree that it makes little sense